app/javascript/packs/application/redux/tours/index.js

```diff
@@ -18,10 +21,12 @@ export const INITIAL_STATE = {
 export const reducer = (state = INITIAL_STATE, action) => {
   switch (action.type) {
     case CREATE_TOUR:
+    case UPDATE_TOUR:
     case LIST_TOURS:
     case GET_TOUR:
       return { ...state, isFetching: true };
     case CREATE_TOUR_SUCCESS:
+    case UPDATE_TOUR_SUCCESS:
```

Comment:
Should these empty success cases return any data?

Reply:
Nope. `switch` statements fall through. So, in this example, she's reusing the same return value for 4 different actions because her actions provide structured data.

Reply:
All of the success cases return the same thing: the existing state merged with any new data received from the api call (the return statement for the successes starts on line 32). 

Am I misunderstanding your question?

Reply:
Nope you got it.  Just didn't see the fall-through case.